maximeish/MyBrand-Maxime
# blog landing page for brand website

#### What does this PR do?
Shows the page to be added to the brand website


#### Description of Task to be completed?
Add a blog landing page to develop branch

#### How should this be manually tested?
Check if all the necessary HTML and CSS files are present, the page should fill the whole page with no scrollbar,
also the Sign up and login buttons should correctly link to their corresponding pages

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate
![blog](https://user-images.githubusercontent.com/54126307/87797871-9409e400-c84b-11ea-839a-37d8fa8dff63.png)


#### Questions:
Why does it show Can't automatically merge error?
![error](https://user-images.githubusercontent.com/54126307/87798022-c582af80-c84b-11ea-8395-7c98880651aa.png)
